packages/zoe/test/unitTests/contracts/gimix/test-postalSvc.js

```diff
@@ -0,0 +1,144 @@
+// @ts-check
+// eslint-disable-next-line import/order
+import { test as anyTest } from '../../../../tools/prepare-test-env-ava.js';
+
+import { createRequire } from 'module';
+
+import { E, Far } from '@endo/far';
+import { unsafeMakeBundleCache } from '@agoric/swingset-vat/tools/bundleTool.js';
+import { makeZoeKitForTest } from '../../../../tools/setup-zoe.js';
+import { makeNameHubKit, makePromiseSpace } from '@agoric/vats';
+import { makeWellKnownSpaces } from '@agoric/vats/src/core/utils.js';
+import { AmountMath } from '@agoric/ertp/src/amountMath.js';
+import { startPostalSvc } from '../../../../src/contracts/gimix/start-postalSvc.js';
+
+const { Fail } = assert;
+
+/** @type {import('ava').TestFn<Awaited<ReturnType<makeTestContext>>>} */
+const test = anyTest;
+
+const myRequire = createRequire(import.meta.url);
+/** @param {string} specifier */
+const asset = specifier => myRequire.resolve(specifier);
+
+/**
+ * Wrap Zoe exo in a Far so that we can override methods.
+ *
+ * @param {ZoeService} zoeMethods
+ */
+const wrapZoe = zoeMethods => {
+  /** @type {ZoeService} */
+  // @ts-expect-error mock
+  const mock = {
+    // XXX all the methods used in this test; there may be others.
+    getFeeIssuer: () => zoeMethods.getFeeIssuer(),
+    getInvitationIssuer: () => zoeMethods.getInvitationIssuer(),
+    installBundleID: (...args) => zoeMethods.installBundleID(...args),
+    startInstance: (...args) => zoeMethods.startInstance(...args),
+    getTerms: (...args) => zoeMethods.getTerms(...args),
+    getPublicFacet: (...args) => zoeMethods.getPublicFacet(...args),
+    offer: (...args) => zoeMethods.offer(...args),
+  };
+  return mock;
+};
+
+const makeTestContext = async t => {
+  const bundleCache = await unsafeMakeBundleCache('bundles/');
+  const bundle = await bundleCache.load(
+    asset('../../../../src/contracts/gimix/postalSvc.js'),
+    'gimix',
+  );
+
+  const bootstrap = async () => {
+    const { produce, consume } = makePromiseSpace();
+
+    const { zoeService } = makeZoeKitForTest();
+    /**@param {string} bid */
+    const install1BundleID = bid => {
+      assert.equal(bid, `b1-${bundle.endoZipBase64Sha512}`);
```

Comment:
```suggestion
    /**@param {string} bID */
    const install1BundleID = bID => {
      assert.equal(bID, `b1-${bundle.endoZipBase64Sha512}`);
```

either "bID" (or "bId") would be fine, but I don't like "bid" as an abbreviation.  Even "b" would be fine, since this is so localized.